src/frontend/src/components/authenticateBox.ts

```diff
@@ -203,16 +202,11 @@ export const authenticate = async (
 
 // Wrap the template with header & footer and render the page
 const page = (slot: TemplateResult) => {
-  const template = html`
-    <div class="l-container c-card c-card--highlight">
-      <!-- The title is hidden but used for accessibility -->
+  const template = mainWindow({
+    slot: html` <!-- The title is hidden but used for accessibility -->
```

Comment:
is this still relevant? alternatively, shouldn't this be applied to all pages or no pages?

Reply:
I don't think so only the ones that don't have title in the content